lib/index.js

```diff
@@ -18,7 +18,8 @@ export type BackupOptions = {|
   prettyPrintJSON: boolean,
   requestCountLimit: number,
   exclude: Array<string>,
-  excludePatterns: Array<RegExp>
+  excludePatterns: Array<RegExp>,
+  fileWrite: void
```

Comment:
Should this also be an optional [function type](https://flow.org/en/docs/types/functions/#toc-function-types)?